src/data_loader.py

```diff
@@ -0,0 +1,319 @@
+"""Module for evaluating Survey Assist SIC/SOC code assignments.
```

Comment:
I would stash this away for now. While it has useful functionality, I think we should start with a more simple approach where data loader is a method under LabelAccuracy (or similar) class.